spec/app/domain/authentication/authn-jwt/restriction_validation/validate_restrictions_one_to_one_spec.rb

```diff
@@ -55,18 +64,32 @@
     Authentication::ResourceRestrictions::ResourceRestriction.new(name: "not_existing", value: "    ")
   }
 
+  let(:mapped_right_restriction) {
+    Authentication::ResourceRestrictions::ResourceRestriction.new(name: "user_login", value: right_login)
+  }
+
+  let(:mapped_wrong_restriction) {
+    Authentication::ResourceRestrictions::ResourceRestriction.new(name: "user_login", value: wrong_login)
+  }
+
+  let(:non_existing_mapped_restriction) {
+    Authentication::ResourceRestrictions::ResourceRestriction.new(name: "machine_name", value: wrong_login)
```

Comment:
I think it will be better to have the right_login here.. just to make sure the failure is on the missing restriction and not on the value

Reply:
Actually it does not matter for non existing restriction.
I'll place other value.